### PR TITLE
add auth0 login on home page

### PR DESCRIPTION
**Pull request by katherinebernstein**

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -19,6 +19,24 @@ class App extends React.Component {
   }
 
   render() {
+    var lock = new Auth0Lock('KvwsETaUxuXVjW2cmz2LbdqXQBdYs6wH', 'loom.auth0.com');
```
Comment (UnsungHero97) [inline src/App.jsx]: please start using ES6 syntax

File: src/index.html [hunk 2]
```diff
@@ -4,8 +4,11 @@
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <title>Gallery</title>
+    <script src="//cdn.auth0.com/js/lock/10.1.0/lock.min.js"></script>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no" />
```
Comment (UnsungHero97) [inline src/index.html]: there's already a `<meta>` tag on line 6. figure out which one is the correct one to use.

File: src/index.html [hunk 3]
```diff
@@ -4,8 +4,11 @@
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <title>Gallery</title>
+    <script src="//cdn.auth0.com/js/lock/10.1.0/lock.min.js"></script>
```
Comment (UnsungHero97) [inline src/index.html]: you don't need this. you should be `import`-ing `Auth0Lock` in `App.jsx`

File: src/index.html [hunk 4]
```diff
@@ -4,8 +4,11 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no" />
   </head>
   <body>
     <div id="app"></div>
+    <button type="submit" id="btn-login">Sign In</button>
```
Comment (UnsungHero97) [inline src/index.html]: this belongs in `App.jsx`

File: src/App.jsx (hunk 1 above)
Comment (UnsungHero97) [inline src/App.jsx]: you'll need something like this...

    import Auth0Lock from 'auth0-lock';

it depends on what they're exporting

File: src/App.jsx [hunk 5]
```diff
@@ -19,6 +19,24 @@ class App extends React.Component {
...
+    var btn_login = document.getElementById('btn-login');
```
Comment (UnsungHero97) [inline src/App.jsx]: this is not the React way to do things. it's probably ok for now to get things going, but it's worth investing a little bit of time to do this the right way

File: src/App.jsx [hunk 6]
```diff
@@ -19,6 +19,24 @@ class App extends React.Component {
...
+    btn_login.addEventListener('click', function() {
+      lock.show();
+    });
+    lock.on("authenticated", function(authResult) {
```
Comment (UnsungHero97) [inline src/App.jsx]: single quotes

File: src/App.jsx (hunk 5 above)
Comment (makosblade) [inline src/App.jsx]: just curious, what is the react way to do this?

Comment (UnsungHero97) [inline src/App.jsx]: I'm not yet an expert, but at the least, we'll have to have a `<Button />` component with an `onClick` listener. to expand on that, it'd probably make sense to have some kind of "login" component or container (these are somehow different), which contains all this lock logic as well as the `<Button />`

Comment (makosblade) [inline src/App.jsx]: Gotcha

**Pull request merged**